src/main/java/Iris.java

```diff
@@ -0,0 +1,56 @@
+import task.Task;
+import static method.Command.*;
+
+import java.util.Scanner;
+import java.util.ArrayList;
+
+public class Iris {
+    static ArrayList<Task> tasks;
+
+    public static void printDivider() {
+        System.out.println("---------------------------------------------");
+    }
+
+    public static boolean chat(String text) {
```

Comment:
It would be good to leave comments on why this method is returning a boolean.
We will discuss comments in Week 7.